src/site/topic9.rst

```diff
@@ -90,6 +90,59 @@ Growth Function
     highlight that it's about change and growth.
 
 
+
+Asymptotic Growth & Big-O
+=========================
+
+* We **don't** care about constants
+* We **don't** care about non dominant terms
+
+* We **do** care about *asymptotic complexity*
+    * Fancy way of saying *The time the algorithm takes as a function of* :math:`n` *will grow like...*
+
+* We care about the *order* of the algorithm
+* We specify the order by using *Big-O notation*
+* In our example, :math:`t(n) = 15n^{2} + 45n` is :math:`O(n^{2})` since it grows like :math:`n^{2}`
+
+
+Example Growth Functions and Their Order
+----------------------------------------
+
++--------------------------------------+------------------------+
+| *Growth Function*                    | *Order*                |
++======================================+========================+
+| :math:`t(n) = 17`                    | :math:`O(1)`           |
++--------------------------------------+------------------------+
+| :math:`t(n) = 20n - 4`               | :math:`O(n)`           |
++--------------------------------------+------------------------+
+| :math:`t(n) = 12n \log_{2}(n) + 100n`| :math:`O(n\log_{2}(n))`|
++--------------------------------------+------------------------+
+| :math:`t(n) = 3n^{2} + 5n - 2`       | :math:`O(n^{2})`       |
++--------------------------------------+------------------------+
+| :math:`t(n) = 2^{2} + 3n`            | :math:`O(2^{n})`       |
```

Comment:
This is just O(n). Presumably `2^{2}` => `2^{n}` on left hand side?

Reply:
Yerp